src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/RadioactivityTask.java

```diff
@@ -0,0 +1,87 @@
+package io.github.thebusybiscuit.slimefun4.implementation.tasks;
+
+import io.github.thebusybiscuit.cscorelib2.chat.ChatColors;
+import io.github.thebusybiscuit.slimefun4.api.player.PlayerProfile;
+import io.github.thebusybiscuit.slimefun4.core.attributes.ProtectionType;
+import io.github.thebusybiscuit.slimefun4.core.attributes.Radioactive;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.implementation.items.RadioactiveItem;
+import io.github.thebusybiscuit.slimefun4.utils.RadiationSymptom;
+import io.github.thebusybiscuit.slimefun4.utils.RadiationUtils;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import net.md_5.bungee.api.ChatMessageType;
+import net.md_5.bungee.api.chat.ComponentBuilder;
+import org.bukkit.Bukkit;
+import org.bukkit.GameMode;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+
+/**
+ * The {@link RadioactivityTask} handles radioactivity for
+ * {@link Radioactive} items.
+ *
+ * @author Semisol
+ *
+ */
+public class RadioactivityTask implements Runnable {
+
+    private final RadiationSymptom[] SYMPTOMS = RadiationSymptom.values();
+
+    @Override
+    public void run() {
+        for (Player p : Bukkit.getOnlinePlayers()) {
+            if (!p.isValid() || p.isDead()) {
+                continue;
+            }
+
+            PlayerProfile.get(p, profile -> handleRadiation(p, profile));
```

Comment:
This is small enough to be simplified

```suggestion
            if (p.isValid() && !p.isDead()) {
                PlayerProfile.get(p, profile -> handleRadiation(p, profile));
            }
```